back-end/controllers/snackController.js

```diff
@@ -0,0 +1,84 @@
+const express = require("express");
+const snacks = express.Router();
+
+const {
+  getAllSnacks,
+  getSnack,
+  createSnack,
+  deleteSnack,
+  updateSnack,
+} = require("../queries/snacks.js");
+
+const { checkName, checkBooleen } = require("../validations/checkSnacks.js");
+const { confirmHealth }= require('../confirmHealth.js')
+// INDEX Route - DONE
+snacks.get("/", async (request, response) => {
+  const allSnacks = await getAllSnacks();
+  if (allSnacks[0]) {
+    response.status(200).json({ payload: allSnacks, success: true });
+  } else {
+    response
+      .status(500)
+      .json({ error: "server error (line 18)! Snack Controller" });
+  }
+});
+
+// SHOW Route - DONE
+snacks.get("/:id", async (req, res) => {
+  const { id } = req.params;
+  const snack = await getSnack(id);
+  if (snack.id) {
+    res.json({payload:snack, success:true});
+
+  } else {
+    res.status(404).json({ payload:'not found', success:false, error:'Show Route Error-Snack Controller Line27'  });
+  }
+});
+
+// NEW Route - DONE
+snacks.post("/", checkName, checkBooleen, async (req, res) => {
+  let name = req.body.name.split()
+  console.log(name)
+
```

Comment:
looks like we started some capitalization functionality on line 40 but never finished.  Consider writing that function and saving it in another file - we could use it again in our edit route